GOVERNANCE.md

```diff
@@ -0,0 +1,69 @@
+### Roles
+
+#### Maintainers
+Maintainers are in charge of the day to day maintenance of the team's projects. They review
+Reviews PRs, ability to merge.
+Ensures code 	quality
+
+New maintainers must already be contributors, must be nominated by an existing maintainer or core team member, and must be elected by a supermajority of the core team. Likewise, maintainers can be removed by a supermajority of the core team or can resign by notifying one of the maintainers.
+
+#### Contributors
+Contributors are those who make regular contributions to the project (documentation, code reviews, responding to issues, participation in proposal discussions, contributing code, etc.). New contributors may be self-nominated or be nominated by existing contributors, and must be elected by a supermajority of that project’s maintainers.
+
+### Implementation Team
+The Implementation team is responsible for maintaining the components that constitute the Cloud Native Buildpacks reference implementation of the specification. This includes:
+
+* [lifecycle](https://github.com/buildpacks/lifecycle)
+* [libbuildpack](https://github.com/buildpacks/libbuildpack)
+
+### Platform Team
+The Platform team is responsible for maintaining the components that constitute the reference platform. This includes:
+
+* [pack](https://github.com/buildpacks/pack)
+* [Tekton Template](https://github.com/tektoncd/catalog/tree/master/buildpacks)
+
+### Learning Team
+The Learning team is responsible for maintaining the project’s documentation, website, and resources. This includes:
+
+* [docs](https://github.com/buildpacks/docs)
+* [resources](https://github.com/buildpacks/resources)
+
+## Emeritus Team
+Serving as a member of an open source project requires a huge amount of work that cannot be sustained indefinitely. The emeritus team includes members of the project to whom we will always be grateful, but who no longer actively participate in the project.
+
+Project members will graduate to the emeritus team with a supermajority vote from the core team, and after giving notice that they no longer intend to participate in the project.
+
+## RFC Process
+Each major decision in Cloud Native Buildpacks starts as a Request for Comments (RFC). Everyone is invited to discuss the proposal, to work toward a shared understanding of the tradeoffs.
+
+RFCs record potential changes with the context and information at the given time. This provides a defined process for anyone wishing to contribute to the Cloud Native Buildpacks project as well as collect a diverse set of inputs and give opportunity for engagement. Anyone who chooses not to actively participate in any RFC is presumed to trust their colleagues on the matter. Once an RFC is accepted, they can be referenced as read-only documents in this repository until replaced or amended by another RFC when context has significantly changed.
+
```

Comment:
Maybe we should call out that RFCs are not to be taken as documentation or specification. Details of the implementation may deviate from the RFC.